tests/tutorial-check/extract_inis.pl

```diff
@@ -16,7 +17,7 @@
 my $nr = 0;
 while (<>) {
   $nr++;
-  if (/code-block::/) {
+  if (/code-block:: ini/) {
```

Comment:
ještě bych byl opatrnej a dal za tu mezeru hvězdičku a místo mezery něco jako \s nebo jak se to řekne